modules/apps/object/object-service/src/main/java/com/liferay/object/internal/layout/tab/screen/navigation/ObjectLayoutTabScreenNavigationCategory.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.internal.layout.tab.screen.navigation;
+
+import com.liferay.frontend.taglib.servlet.taglib.ScreenNavigationCategory;
+import com.liferay.object.model.ObjectLayoutTab;
+
+import java.util.Locale;
+
+/**
+ * @author Feliphe Marinho
+ */
+public class ObjectLayoutTabScreenNavigationCategory
+	implements ScreenNavigationCategory {
+
+	public ObjectLayoutTabScreenNavigationCategory(
+		ObjectLayoutTab objectLayoutTab) {
+
+		_objectLayoutTab = objectLayoutTab;
+	}
+
+	@Override
+	public String getCategoryKey() {
+		return String.valueOf(_objectLayoutTab.getObjectLayoutTabId());
+	}
+
+	@Override
+	public String getLabel(Locale locale) {
+		return _objectLayoutTab.getName(locale);
+	}
+
+	@Override
+	public String getScreenNavigationKey() {
```

Comment:
@MarinhoFeliphe This value must be unique. We should follow some pattern like {className}#{LayoutERC} or {className}#{companyId}#{LayoutERC}.

P.S.: We should check if the tracker of this is scoped by company or not. I don't think so.

Reply:
@MarinhoFeliphe can you open a thread with he FI Team so we can ask about scoping issues (regarding company).

Reply:
@gabrielwas,

we don't have ERC for ObjectLayout yet, may I use layoutId?

> can you open a thread with he FI Team ...

sure!